chains/manager.go

```diff
@@ -1393,11 +1396,14 @@ func (m *manager) StartChainCreator(platformParams ChainParameters) error {
 	m.createChain(platformParams)
 
 	m.Log.Info("starting chain creator")
+	m.chainCreatorExited.Add(1)
```

Comment:
Normally I'd use an exit channel for this given the wg will only ever have value 1?

Reply:
looking around in avalanchego - I think it's much more common for us to use a wg than a channel for this. Even with a single goroutine